monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -433,7 +561,7 @@ def train_parallel_cmd(
         from nnunetv2.training.dataloading.utils import unpack_dataset
 
         for folder_name in folder_names:
-            logger.warning(f"[info] unpacking '{folder_name}'...")
+            logger.warning(f"unpacking '{folder_name}'...")
```

Comment:
do we have other options than `warning` for the printing messages?

Reply:
Here I would suggest info level @dongyang0122

Reply:
Fixed in commit [4a7136a](https://github.com/Project-MONAI/MONAI/pull/6448/commits/4a7136adc408c0f860cf3be4614d79ac7589bcfe)